### PR TITLE
New tests

### PR DESCRIPTION
**Pull request by wlodarqa**

### CODE REVIEW

File: cypress/integration/tests/login.spec.js [hunk 1]
```diff
@@ -1,21 +1,35 @@
 describe('Login tests ', () => {
```
Comment (joszo) [inline cypress/integration/tests/login.spec.js:1]: remove space after 'tests'

File: cypress/integration/tests/login.spec.js [hunk 2]
```diff
@@ -1,21 +1,35 @@
...
+    it('login with invalid data', () => {
+        cy.get('#username').type('wrong name')
+        cy.get('#password').type('wrong password')
+        cy.get('[type="checkbox"]').check()
+        cy.get('.MuiButton-label').click()
+        cy.get('.MuiAlert-message').should('be.visible')
+        })
+
+        it('login without checkbox', () => {
+            cy.get('#username').type('wlodarqa')
```
Comment (joszo) [inline cypress/integration/tests/login.spec.js:29]: wrap those 3 lines into custom commands

File: cypress/integration/tests/login.spec.js [hunk 3]
```diff
@@ -1,21 +1,35 @@
...
+            cy.get('#password').type('test')
+            cy.get('.MuiButton-label').click()
+            cy.wait(2000)
```
Comment (joszo) [inline cypress/integration/tests/login.spec.js:32]: why do we need that wait?

File: cypress/integration/tests/login.spec.js [hunk 4]
```diff
@@ -1,21 +1,35 @@
...
-     })
+        cy.get('.makeStyles-root-1').should('be.visible')
+        })
+
+    it('login with invalid data', () => {
+        cy.get('#username').type('wrong name')
+        cy.get('#password').type('wrong password')
+        cy.get('[type="checkbox"]').check()
+        cy.get('.MuiButton-label').click()
+        cy.get('.MuiAlert-message').should('be.visible')
```
Comment (joszo) [inline cypress/integration/tests/login.spec.js:25]: add verification of login request

cy.wait('@login').then(intercept => {})

File: cypress/integration/tests/login.spec.js (hunk 1 above)
Comment (wlodarqa) [reply cypress/integration/tests/login.spec.js:1]: done

File: cypress/integration/tests/login.spec.js (hunk 3 above)
Comment (wlodarqa) [reply cypress/integration/tests/login.spec.js:32]: This test is deleted

File: cypress/integration/tests/login.spec.js (hunk 1 above)
Comment (joszo) [reply cypress/integration/tests/login.spec.js:1]: Did you pushed the newest version?

File: cypress/integration/tests/login.spec.js (hunk 3 above)
Comment (joszo) [reply cypress/integration/tests/login.spec.js:32]: Why do you want to delete this test? You could verify if the checkbox is really send during the login process.

File: cypress/integration/tests/login.spec.js (hunk 4 above)
Comment (wlodarqa) [reply cypress/integration/tests/login.spec.js:25]: Can I add it to beforeEach?

File: cypress/integration/tests/login.spec.js (hunk 3 above)
Comment (wlodarqa) [reply cypress/integration/tests/login.spec.js:32]: Ok, I have a new idea for this test.

**Pull request merged**